src/toniesJson.c

```diff
@@ -34,6 +34,15 @@ void tonies_init()
     }
 }
 
+void *tonies_updatePeriodically(void *arg)
+{
+    while (true)
+    {
+        tonies_update();
```

Comment:
Please run the first tonies_update on startup synchros to the rest, then also check if the settings for updating is active

get_settings()->core.tonies_json_auto_update